autopilot/contracts.go

```diff
@@ -39,3 +77,244 @@ func (ap *Autopilot) defaultContracts() ([]worker.Contract, error) {
 	}
 	return contracts, nil
 }
+
+func (ap *Autopilot) contractLoop() {
+	for {
+		select {
+		// TODO: use build var for loop interval
+		// TODO: add wakeChan (to trigger loop)
```

Comment:
Do we need to trigger the loop? Is it not good enough if it runs every x minutes?

Reply:
Figured we would need it for whenever the autopilot config is updated or for whenever we sync up we could trigger it.

Reply:
resolving for now